test_scripts/API/Restructuring_OnResetTimeout/019_BC_OnResetTimeout_reseting_is_not_applied_when_remaining_time_from_current_timeout_is_greater.lua

```diff
@@ -0,0 +1,44 @@
+------------------------------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0189-Restructuring-OnResetTimeout.md
+------------------------------------------------------------------------------------------------------------------------
+-- Description: Check SDL is able to ignore reset period received in 'OnResetTimeout()' notification in case
```

Comment:
Can you please point to specification for this?

Reply:
@atiwari9 There is no specific requirement for this scenario. However in this case we assumed Core has to ignore the value of `resetPeriod` if its lower than default timeout (10s). Or our assumption was incorrect?

Reply:
No it should still take effect, and as per my manual tests with `SDL_HMI` and `SDL_Core` PRs for this feature, core does reset the timeout to lower value as well. Please update the test case.